resources/recipes.py

```diff
@@ -229,6 +229,7 @@ def plank(thing: str):
     rm.crafting_shaped('crafting/vanilla/bowl', ['XYX', ' X '], {'X': '#tfc:lumber', 'Y': 'tfc:glue'}, (3, 'minecraft:bowl')).with_advancement('#tfc:lumber')
     rm.crafting_shaped('crafting/vanilla/scaffolding', ['XYX', 'X X', 'X X'], {'X': 'minecraft:bamboo', 'Y': '#forge:string'}, (6, 'minecraft:scaffolding')).with_advancement('minecraft:bamboo')
     rm.crafting_shaped('crafting/vanilla/bow', [' YX', 'Y X', ' YX'], {'X': '#forge:rods/wooden', 'Y': '#forge:string'}, 'minecraft:bow').with_advancement('#forge:string')
+    rm.crafting_shaped('crafting/vanilla/bow2', ['XY ', 'X Y', 'XY '], {'X': '#forge:rods/wooden', 'Y': '#forge:string'}, 'minecraft:bow').with_advancement('#forge:string')
```

Comment:
This is misunderstanding the issue. It is not one of mirroring (recipes will mirror themselves automatically.) The ingredients are swapped.

Vanilla
![minecraft](https://user-images.githubusercontent.com/35673674/197411205-557d7999-1d26-487b-ba93-80b9597197ac.png)

TFC
![tfc](https://user-images.githubusercontent.com/35673674/197411346-3107878a-778d-48a6-8ea7-5f0e8da8a75e.png)